app/src/hooks/eventHooks.ts

```diff
@@ -27,19 +27,38 @@ export const useEvents = () => {
   );
 };
 
-export const useCreatedEvents = (): {
-  createdEventIds: string[];
-  setCreatedEventId: (string: string) => void;
+type EditEventToken = {
+  eventId: string;
+  editToken: string;
+};
+const isCreatedEvent = (x: any): x is EditEventToken =>
```

Comment:
isValidEvent? Synes det høres litt weird ut med isCreatedEvent, alle events er jo created på et tidspunkt? evt hasCreatedEvent, om editToken finnes for brukeren liksom?